test_ptrack/settings.py

```diff
@@ -41,7 +34,7 @@
     }
 ]
 
-ALLOWED_HOSTS = ['localhost']
+ALLOWED_HOSTS = ['localhost', 'testserver']
```

Comment:
~~This change was required after switching from `nose` to `pytest`. This might be specific to how `pytest-django` executes tests.~~ (Edit: It appears this change is no longer needed. Maybe due to the removal of django-webtest?)